app/controllers/collections_controller.rb

```diff
@@ -47,37 +47,37 @@ def start_new_search_session?
     has_search_parameters?
   end
 
-  def find_channel
-    Channel.find_by_key!(params[:id]).tap do |channel|
-      authorize! :show, channel
+  def find_collection
+    Collection.find_by_key!(params[:id]).tap do |collection|
+      authorize! :show, collection
     end
   end
 
   def find_landing_page
-    Page::Landing.find_or_initialize_by(slug: "channels/#{@channel.key}").tap do |landing_page|
+    Page::Landing.find_or_initialize_by(slug: "collections/#{@collection.key}").tap do |landing_page|
       authorize! :show, landing_page
     end
   end
 
   ##
-  # Gets from the cache the number of items of each media type within the current channel
-  def channel_stats
+  # Gets from the cache the number of items of each media type within the current collection
+  def collection_stats
```

Comment:
Assignment Branch Condition size for collection_stats is too high. [16.28/15]<br>Method has too many lines. [13/10]